src/commands/Minion/inferno.ts

```diff
@@ -597,6 +597,27 @@ AND (data->>'diedPreZuk')::boolean = false;`)
 			realDuration
 		} = res;
 
+		let boosts = `
+**Boosts:** ${duration.messages.join(', ')} ${
+			duration.missed.length === 0 ? '' : `*(You didn't get these: ||${duration.missed.join(', ')}||)*`
+		}
+**Range Attack Bonus:** ${usersRangeStats.attack_ranged}`;
+		let deathChances = `
+**Pre-Zuk Death Chance:** ${preZukDeathChance.value.toFixed(1)}% ${preZukDeathChance.messages.join(', ')} ${
+			preZukDeathChance.missed.length === 0
+				? ''
+				: `*(You didn't get these: ||${preZukDeathChance.missed.join(', ')}||)*`
+		}
+**Zuk Death Chance:** ${zukDeathChance.value.toFixed(1)}% ${zukDeathChance.messages.join(', ')} ${
+			zukDeathChance.missed.length === 0
+				? ''
+				: `*(You didn't get these: ||${zukDeathChance.missed.join(', ')}||)*`
+		}`;
+
+		if (msg.flagArgs.chance) {
+			return msg.channel.send(`${boosts}${deathChances}`);
```

Comment:
This should probably be in the base command (`+inferno`) instead of being a flag, in the command to start one?

Reply:
I can move it into the base command but it would mean also copying up the code to both do an inferno run and parse/format the boosts and chances returned.

Since the flag is only intended to show the death chances, not the boosts (it was just there and nice to have), I will move the flag to the base command and call the death chance calculations from there.

Reply:
Nevermind, looking at the code I'd be skipping all of the gear calculations that affect death chance by doing this.

I'll move it as you suggested and we can re-review.